script/predicate_functions.sh

```diff
@@ -23,6 +23,18 @@ function is_mri_192 {
   fi
 }
 
+function is_mri_192_plus {
+  if is_mri; then
+    if ruby -e "exit(RUBY_VERSION.to_f > 1.9)"; then
```

Comment:
This conditional isn't right.  It returns `false` on 1.9.2 and 1.9.3.  You need `RUBY_VERSION.to_f >= 1.9` or `RUBY_VERSION.to_f > 1.8` to have it return true on 1.9.2 and 1.9.3.